RallyHereIntegration/Source/RallyHereIntegration/Private/RH_EntitlementSubsystem.cpp

```diff
@@ -144,21 +144,22 @@ IOnlineSubsystem* URH_EntitlementSubsystem::GetOSS() const
 			return nullptr;
 		}
 
-		IOnlineSubsystem* oss = LPSS->GetOSS(EntitlementOSSName);
-		if (oss == nullptr)
+		IOnlineSubsystemPtr OSS = MakeShareable(LPSS->GetOSS(EntitlementOSSName));
```

Comment:
Doesn't `MakeShareable` assume ownership of the pointer - and thus delete it once it is out of scope?

Reply:
After further review this does appear to be more risky than I was assuming.  Without being able to use a shared pointer, I am going to use a different approach in a subsequent PR.